public/js/ajax-airline-handler.js

```diff
@@ -0,0 +1,120 @@
+$('form').submit(function(event){
+    event.preventDefault();
+})
+
+const createAirline = () => {
+    fetch('api/airlines', {
+        method: 'POST',
+        headers: {
+            'Content-type': 'application/x-www-form-urlencoded',
+            'X-Requested-With': 'XMLHttpRequest',
+            'X-CSRF-TOKEN': getCsrfToken()
+        },
+        body: new URLSearchParams($('#add-airline-form').serialize())
+    })
+
+    .then(response => {
+        if (response.ok) {
+            $('#name-error').text('');
+            $('#description-error').text('');
+            document.forms["add-airline-form"].reset();
+
+            return response.json();
+
+        }
+        return response.json().then(data => {
+            handleValidationErrors(data.errors);
+            throw new Error('Validation Error');
+        });
+    })
+    .then(result => loadTable())
+    .catch(error => console.error);
+}
+
+const deleteAirline = (airlineId) => {
+    fetch(`api/airlines/${airlineId}`, {
+        method: 'DELETE',
+        headers: {
+            'X-Requested-With': 'XMLHttpRequest',
+            'X-CSRF-TOKEN': getCsrfToken()
+        },
+    })
+    .then(response => response.json())
+    .then(result => loadTable())
+    .catch(error => console.error)
+}
+
+const updateAirline = (airlineId) => {
+    fetch(`api/airlines/${airlineId}`, {
+        method: 'PUT',
+        headers: {
+            'Content-type': 'application/x-www-form-urlencoded',
+            'X-Requested-With': 'XMLHttpRequest',
+            'X-CSRF-TOKEN': getCsrfToken()
+        },
+        body: new URLSearchParams($('#airlines-update-form').serialize())
+    })
+
+    .then(response => response.json())
+    .then(result => loadTable())
+    .catch(error => console.error);
+}
+
+const loadTable = () => {
+    const page = currentPage();
+    fetch(`/api/airlines?page=${page}`)
+
+    .then(response => response.json())
+    .then(result => {
+        const airlines = result.data;
+        $('#table-body').html(renderTable(airlines))
```

Comment:
```suggestion
        $('#table-body').html(renderTable(result.data))
```
You don't really need the `airline` cost if all you're gonna do is use if in a single place.